deploy/template.yaml

```diff
@@ -2410,11 +2492,22 @@ Resources:
           AttributeType: "S"
         - AttributeName: "credentialIssuer"
           AttributeType: "S"
+        - AttributeName: "dateCreated"
+          AttributeType: "S"
       KeySchema:
         - AttributeName: "userId"
           KeyType: "HASH"
         - AttributeName: "credentialIssuer"
           KeyType: "RANGE"
+      GlobalSecondaryIndexes:
```

Comment:
Are we still keeping this?

Reply:
Nope, removed

Reply:
I think you removed it before I'd even submitted the comment :D